src/index.js

```diff
@@ -1 +1,49 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+let collection = [
+  {
+    description: 'Reading',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Preparation for test',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Project',
+    completed: false,
+    index: 3,
+  },
+];
+
+localStorage.setItem('todoList', JSON.stringify(collection));
+
+function ShowList(arr) {
+  const listToDo = arr.map((b) => `
+    <ul class="testList1" draggable="true">
+          <li><input type="checkbox" id='${b.index}' value='${b.completed}' class="checkboX">
+          <p>${b.description}</p>
+          </li>
+          <i class=" fa fa-ellipsis-v" id='${b.index}'></i>
+          <i class="fa fa-trash-o" id='${b.index}'></i>          
+      </ul>
```

Comment:
- [ ] OPTIONAL - Consider changing this regular function to es6 arrow function. As stated in the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.).